docs/pages/user-docs/profile-page.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title: Profile Page
+parent: User Guide
+nav_order: 630
+---
+# :two_women_holding_hands: Profiles of colleagues 
+The profiles list contact details, direct supervisor, team assistant, profit center, employment type, organizational unit, and location. 
+
+In the lower part of the profile you will also find the map to the colleagues office.  
+
+You can choose from the following actions:
+
+* **Contact directly**: Click on the email address to open an email or click on the phone number to start a call. 
+* **Open Map**: By clicking "show room" you open the map navigated to the room of your colleague. Now you can also see who else is sitting in this room and where it's placed in general.  
+* **Open Organigram**: By clicking "show organigram" you open the organigram at the exact position your colleague is located at. This offers you insight into the other team members and their corresponding supervisors.  
+* **Share profile**: Use ![share icon](media/shareicon.png) to either download the whole profile, share it via Rocketchat, copy the link to the current profile or share it via mail. 
```

Comment:
You mean if you download the vcard? With the shared link can you see the image. 

I would not right this to this point because it's not really necessary and not an important fact for the user, i think

Reply:
because the download is explicitly described above, I would create a little warning so the user knows that this isn't a bug.

Reply:
It is a not yet implemented feature, there is no reason to not include it.

Reply:
oh boy, double negative. Can one of you just write the phrase you'd want to have included as a comment? I'll add it

Reply:
Just leave it as it is :)